content/es/iot/index.html

```diff
@@ -65,14 +65,17 @@
     <li>Velocidad de transferencia de datos de 100 bit/s</li>
 </ul>
 
-<p>Actualidad de Sigfox en República Checa
-    <img src="https://www.simplecell.eu/wp-content/uploads/2017/01/coverage-cz-170117.jpg" alt="Mapa pokrytí ČR sítě Sigfox" class="center">
+<p>Actualidad de Sigfox en España
+    <img src="/img/iot/map-sigfox-es.png" alt="Mapa de cobertura Sigfox" class="center">
 </p>
 
 
 <h2>LoRa</h2>
-<p>Desde el 1 de abril del año 2016, la red LoRa opera en las radiocomunicaciones 
-de la República Checa. Disponible en todas las capitales regionales.</p>
+
+<p>Desde el 1 de abril del año 2016, la red LoRa opera en las radiocomunicaciones
+de España. Disponible en todas las capitales regionales.</p>
```

Comment:
I assume this paragraph says that LoRa is available since 2016 but the next one says that it's not available at all.